### DISCUSSION TITLE
comment je peux annulet beta dans github

### DISCUSSION THREAD

**Discussion by Spako46** [community/community#55076] [Accessibility]
### Select Topic Area

Question

### Body

comment je peux annuler beta dans github ?

**Comment by ghostinhershell** [comment]
Hi @Spako46 , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

With that being said, you may find the answer to your question in our GitHub Support Docs here: https://docs.github.com/en.

Cheers!
